cmd/opentelemetry/app/defaultconfig/default_config.go

```diff
@@ -206,13 +206,15 @@ func createExporters(component ComponentType, storageTypes string, factories com
 
 func enableAgentUDPEndpoints(jaeger *jaegerreceiver.Config) {
 	if jaeger.ThriftCompact == nil {
-		jaeger.ThriftCompact = &confignet.TCPAddr{
+		jaeger.ThriftCompact = &jaegerreceiver.ProtocolUDP{
```

Comment:
Nice.  This will allow us to support max udp packet size in the Jaeger otel collectors.

Reply:
Yeah, it is already configurable in otel receiver config

Reply:
Alongwith socket buffer size, number of workers and queue size